go/oasis-node/cmd/registry/runtime/runtime.go

```diff
@@ -537,7 +537,7 @@ func init() {
 	runtimeFlags.Uint64(CfgExecutorGroupSize, 1, "Number of workers in the runtime executor group/committee")
 	runtimeFlags.Uint64(CfgExecutorGroupBackupSize, 0, "Number of backup workers in the runtime executor group/committee")
 	runtimeFlags.Uint64(CfgExecutorAllowedStragglers, 0, "Number of stragglers allowed per round in the runtime executor group")
-	runtimeFlags.Duration(CfgExecutorRoundTimeout, 10*time.Second, "Executor committee round timeout for this runtime")
+	runtimeFlags.Int64(CfgExecutorRoundTimeout, 5, "Executor committee round timeout for this runtime (in consensus blocks)")
```

Comment:
Maybe add an extra note to the `.changelog/3210.breaking.md`:

> Note: Existing deployments will need to alter the state dump and update the executor round timeout in all registered compute runtimes.

Reply:
Good idea, added.